documentation/api/reference/made_with_ov.md

```diff
@@ -83,3 +83,16 @@ to power its [image search](https://www.mojeek.com/images) feature.
 **Available media:** images
 
 ![Mojeek image search](/_static/openverse_mojeek.png)
+
+## Sutori
+
+[Sutori](https://www.sutori.com/) is a collaborative instruction and
+presentation tool for the classroom. Sutori integrates with Openverse to provide
+direct access to our vast catalog of images from their media uploader tool.this
+is the them annouceing the intergrateion
+
+**Available media:** images
+
+![Mojeek image search](/_static/openverse_sutori.png)
+
+[Sutori integration](https://help.sutori.com/en/articles/6825001-release-notes-may-2022)
```

Comment:
Overall, looks good to me, but we'll need to tidy this up a bit. 

The `Sutori integration` link leads to a 404 currently. @dhruvkb shared this link (<https://headwayapp.co/sutori-updates/millions-of-free-educational-resources-299145>) in the issue, which we should use instead.

We also need to fix the last sentence, and avoid using the ambiguous "this" article (see my suggestion below). The alt-text for the image also needs to be corrected. See my suggestion for that as well.

```suggestion
direct access to our vast catalog of images from their media uploader tool.

[Refer to Sutori's announcement of Openverse image search integration for further details](https://headwayapp.co/sutori-updates/millions-of-free-educational-resources-299145)

**Available media:** images

![Openverse image search in Sutori](/_static/openverse_sutori.png)
```